lms/templates/sysadmin_dashboard.html

```diff
@@ -164,7 +164,7 @@ <h3>${_('Administer Courses')}</h3><br/>
 %endif
 
 %if msg:
-    <p>${msg}</p>
+    <p>${msg | n, decode.utf8}</p>
```

Comment:
If i remember, some of the tests were failing because of page level escaping, so i had to remove the escaping filter.

Reply:
I don't like this given the combo of:
1. We aren't sure of the messages, and
2. This is a sys admin dashboard, which makes it more important that it is right.

If you can see where msg is coming from, maybe you can properly escape in the Python code using Text/HTML?  Or, if the test was failing simply because an `&` needed to be an `&amp;`, or something like that, maybe just update the test?